tools/sambamba/.shed.yml

```diff
@@ -1,10 +1,10 @@
 name: sambamba
-owner: rnateam
-description: "SAMBAMBA: process your BAM data faster!"
-homepage_url: https://github.com/lomereiter/sambamba
+owner: bgruening
```

Comment:
we need to keep it at rnateam, otherwise it will get a different tool-ID and with this it is a different tool in the Galaxy context